src/__screenshot_tests__/private-responsive-layout-scenarios-screenshot-test.tsx

```diff
@@ -0,0 +1,13 @@
+import {openStoryPage} from '../test-utils';
+
+const DEVICES = ['MOBILE_IOS', 'DESKTOP', 'LARGE_DESKTOP'] as const;
+
+test.each(DEVICES)('ResponsiveLayout scenarios (%s)', async (device) => {
+    const page = await openStoryPage({
+        id: 'private-responsivelayout-scenarios--default',
+        device,
+    });
+
+    const image = await page.screenshot({fullPage: true});
+    expect(image).toMatchImageSnapshot();
+});
```

Comment:
I think I'd create a different screenshot for each scenario. Otherwise it's difficult to know what are we testing.
Also, I'd include some text in the screenshot describing the case under test

Reply:
Done